package.json

```diff
@@ -280,5 +279,9 @@
         "scroll-into-view-if-needed": "^1.5.0",
         "tabbable": "^1.1.2",
         "uuid": "^3.3.2"
+    },
+    "dependencies": {
```

Comment:
should these be dev deps?
Normally we dont add any deps, only dev deps and/or peer deps

Reply:
Also I am not sure if we should be adding dependency to libraries which dont have wide support.

Reply:
I can move them to to peer dependencies.

These libraries are ones that I wrote so that the babel plugin and this component share the way that they convert a tree of components and strings into a single translatable string and back.

Reply:
Question: now the CI build fails because the peer dependencies for my new modules are not there in the travis node_modules cache. How do I update that cache with those new modules?

Reply:
Add peer deps to dev dependencies as well if they need to be run in tests

Reply:
So unfortunately we can't add libraries that folks wrote on their own. We want to only use libraries that have wide community support for future maintenance. They should either be made part of this commit, so that they are reviewed or the functions extracted out here. We can't assume that some library will always be maintained when someone leaves box. Have had problems in the past.

Reply:
Seems like we're at a bit of an impasse with the dependencies. What are our next steps here, @priyajeet?